packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -261,9 +261,30 @@ export class ProposedTransformationExplorer {
             treeDataProvider: transformDataProvider,
         })
 
+        const reset = async () => {
+            await vscode.commands.executeCommand('setContext', 'gumby.transformationProposalReviewInProgress', false)
+            await vscode.commands.executeCommand('setContext', 'gumby.reviewState', TransformByQReviewStatus.NotStarted)
+
+            // delete result archive after changes cleared
+            // Summary is under ResultArchiveFilePath
```

Comment:
nit: maybe add a comment saying that `force: true` prevents an error from being thrown if the file path does not exist, since this `reset()` is called at beginning of every job where those file paths won't always exist